erpnext/accounts/report/item_wise_purchase_register/item_wise_purchase_register.py

```diff
@@ -293,10 +293,18 @@ def get_conditions(filters):
 		("from_date", " and `tabPurchase Invoice`.posting_date>=%(from_date)s"),
 		("to_date", " and `tabPurchase Invoice`.posting_date<=%(to_date)s"),
 		("mode_of_payment", " and ifnull(mode_of_payment, '') = %(mode_of_payment)s"),
-		("item_group", " and ifnull(`tabPurchase Invoice Item`.item_group, '') = %(item_group)s"),
 	):
 		if filters.get(opts[0]):
 			conditions += opts[1]
+	if filters.get("item_group"):
```

Comment:
@Nandhinidevi123 A better way to do this would be https://github.com/frappe/erpnext/blob/develop/erpnext/accounts/report/financial_statements.py#L591

Also, the code is not formatted, please setup linters locally by referring https://github.com/frappe/erpnext/wiki/Pull-Request-Checklist#linting and then reopen the PR